cloud/google/gceproviderconfig/types.go

```diff
@@ -30,4 +30,15 @@ type GCEProviderConfig struct {
 
 	// The name of the OS to be installed on the machine.
 	OS string `json:"os"`
+	Image       string `json:"image"`
+	Disks       []Disk `json:"disks"`
+}
+
+type Disk struct {
```

Comment:
Since you are modifying the the gce provider config, remember to bump all relevant images after merge. eg. the gce-machine-actuator image.

Reply:
Thanks for the reminder -- I was not aware of this.